bundles/org.openhab.binding.vallox/README.md

```diff
@@ -0,0 +1,165 @@
+| postHeatingOn             | Switch |  | Post heating active |
+| preHeatingOn              | Switch |  | Pre heating active |
+| postHeatingTargetValue    | Number |  | Post heating target value of incoming air |
+| damperMotorPosition       | Switch |  | Damper motor position |
+| firePlaceBoosterSwitch    | Switch |  | Booster switch |
+| incomingCurrent           | Number |  | Incoming current from remote control. Value is milliampere (mA) |
+| slaveMasterIndicator      | Switch |  | Slave/master selection |
+| firePlaceBoosterOn        | Switch |  | Booster activated|
+| firePlaceBoosterCounter   | Number |  | Remaining minutes until booster turns to OFF |
+| remoteControlOn           | Switch |  | Remote control acticated |
+| **Maintenance**      
+| filterGuardIndicator          | Switch |  | Filter guard active. Check filters |
+| serviceReminderIndicator      | Switch |  | Service reminder. Do service |
+| maintenanceMonthCounter       | Number |  | Shows how many months to next service |
+| serviceReminder               | Number | ✓ | How many months between services |
+| **Alarm**       
+| co2Alarm                      | Switch |  | CO2 alarm active |
+| faultIndicator                | Switch |  | Fault active |
+| faultSignalRelayClosed        | Switch |  | Fault relay is closed |
+| hrcFreezingAlarm              | Switch |  | Heat recovery cell is freezing |
+| lastErrorNumber               | Switch |  | Number of last error |
+| waterRadiatorFreezingAlarm    | Switch |  | Water radiator is freezing |
+        
+
+
+## Examples
+
+### Thing definitions
+
+#### TCP/IP
+
+```
+Thing vallox:se-tcp:main [ tcpHost="192.168.0.57", tcpPort=26, panelNumber=5 ]
+```
+
+#### Serial
+
+Under Windows use normal com port names e.g. "COM3":
+
+```
+Thing vallox:se-serial:main [ serialPort="/dev/ttyUSB0", panelNumber=5 ]
+Thing vallox:se-serial:main [ serialPort="COM3", panelNumber=8 ]
+```
+
+
+### Item definition
+
+```
+Number FanSpeed          "Fan speed"          { channel="vallox:se-tcp:vallox:fanControl#fanSpeed" } 
+Number TempInside        "Inside temperature" { channel="vallox:se-tcp:vallox:temperature#tempInside" }
+Number AverageEfficiency "Average efficiency" { channel="vallox:se-tcp:vallox:efficiency#averageEfficiency" }
+Switch PowerState        "Power state"        { channel="vallox:se-tcp:vallox:setting#powerState" }
+
+
+Number CO2               "Measured CO2 level                     { channel="vallox:se-serial:main:status#co2" }
+Number CO2SetPoint       "CO2 setpoint for automatic adjustment" { channel="vallox:se-serial:main:setting#co2SetPoint" }
+
+Switch ServiceReminderIndicator "Service reminder"  { channel="vallox:se-tcp:vallox:maintenance#serviceReminderIndicator" }
+Switch FaultIndicator           "Fault indicator"   { channel="vallox:se-tcp:vallox:alarm#faultIndicator" }
+Number LastErrorNumber          "Last error number" { channel="vallox:se-tcp:vallox:alarm#lastErrorNumber" }
+```
+
+
+## Troubleshooting
```

Comment:
AFAIK this should not be part of this README, because it's not specific to this binding

Reply:
Removed.